packages/web-components/fast-components/src/design-tokens.ts

```diff
@@ -291,6 +291,23 @@ export const neutralStrokeDividerRestDelta = create<number>(
     "neutral-stroke-divider-rest-delta"
 ).withDefault(8);
 
+/** @public */
+export const neutralStrokeInputFilledRestDelta = create<number>(
+    "neutral-stroke-input-filled-rest-delta"
+).withDefault(4);
+/** @public */
+export const neutralStrokeInputFilledHoverDelta = create<number>(
+    "neutral-stroke-input-filled-hover-delta"
+).withDefault(10);
+/** @public */
+export const neutralStrokeInputFilledActiveDelta = create<number>(
+    "neutral-stroke-input-filled-active-delta"
+).withDefault(10);
+/** @public */
+export const neutralStrokeInputFilledFocusDelta = create<number>(
+    "neutral-stroke-input-filled-focus-delta"
+).withDefault(49);
```

Comment:
I see this token isn't used, but this is really large for an offset such that it would cap out at black or white for most input colors. This isn't the actual focus ring color, but represents any stroke that might still be visible on a control if the focus is drawn outside, like we do for checkboxes, etc. Often times the focus state is the same as the rest state, which may be better here unless there's another place you're using this token and it needs to be very different from the background.

Reply:
updated